_documentation/user/flags/steamroller.md

```diff
@@ -5,4 +5,26 @@ type: Good
 summary: Destroys tanks you touch but you have to get really close.
 ---
 
-...
+Steamroller flag allows the player to kill tanks that get in a specified proximity of the holder.
+
+## Potency
+
+The effect of killing the opponent automatically if they get close enough can be either really powerful or weak. In the case of Steamroller, the prime factor that decides that is the sole server variable the flag has; *srRadiusMult*.
+
+## Server variables
+
+### Usage and effect
+
+Steamroller has a relatively straightforward effect, but its usefulness can vary depending on how far the automatic killing of the opponents can reach, which is declared through the *srRadiusMult* server variable.
+
+Having a low radius of effect will make the flag commonly reach the levels of uselessness, especially when in a battle of shooting tanks. Having a high radius, on the other hand, will make direct 1-on-1 battles much more successful, as tanks will approach each other and when close enough, any sort of wrong move by the opponent will result in them getting steamrolled.
+
+The effect of the high radius also comes useful when attempting to kill opponents with the [Burrow](../burrow/)</a> flag, as the range of reach can increase, killing such opponents sooner than normally possible.
+
+### Caution
+
+High radius or sudden grabs of the Steamroller flag while in the proximity of teammates can result in unwanted teamkills, should the server configuration not include the prevention of teamkills.
+
+### Associated variables
```

Comment:
how does this section work with Server Variables? I guess these are all sub-sections, but that makes the structure odd. Usage and Effect, Caution, and Associated Variables don't seem to belong as subordinate to Server Variables

(by the way, shouldn't all the section headings be Title Case?)

Reply:
I agree with this comment. I'd prefer a structure of:

- Potency
- Usage and Effect
  - Caution
- Per Server Settings (title tbd)

The server settings section should specify that the settings may change on a per-server basis and should provide some brief detail to server owners or map makers on how to configure it. But the bulk of the detail should remain on the dedicated BZDB page when it comes time to document that.